packages/styled-components/src/constructors/test/createGlobalStyle.test.js

```diff
@@ -332,7 +317,7 @@ describe(`createGlobalStyle`, () => {
     );
 
     expect(console.warn.mock.calls[0][0]).toMatchInlineSnapshot(
-      `"The global style component sc-global-2176982909 was given child JSX. createGlobalStyle does not render children."`
+      `"The global style component sc-global-a was given child JSX. createGlobalStyle does not render children."`
```

Comment:
Canary switched `GlobalStyle` over to use the same name generator, hence we don't see hashes here or in other snapshots anymore